<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Setting up reports and Test is getting ready]]>
    </line>
    <line>
      <![CDATA[Setting Done- Test can be started]]>
    </line>
    <line>
      <![CDATA[Trying to start Browser and Getting application ready]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="197" started-at="2019-11-15T18:29:43 IST" finished-at="2019-11-15T18:29:43 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="197" started-at="2019-11-15T18:29:43 IST" finished-at="2019-11-15T18:29:43 IST">
      <class name="com.learnautomation.testcases.LoginTestCRM">
        <test-method status="PASS" signature="setUpSuite()[pri:0, instance:com.learnautomation.testcases.LoginTestCRM@3b084709]" name="setUpSuite" is-config="true" duration-ms="2381" started-at="2019-11-15T18:29:41 IST" finished-at="2019-11-15T18:29:43 IST">
          <reporter-output>
            <line>
              <![CDATA[Setting up reports and Test is getting ready]]>
            </line>
            <line>
              <![CDATA[Setting Done- Test can be started]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method status="FAIL" signature="setup()[pri:0, instance:com.learnautomation.testcases.LoginTestCRM@3b084709]" name="setup" is-config="true" duration-ms="20" started-at="2019-11-15T18:29:43 IST" finished-at="2019-11-15T18:29:43 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.learnautomation.utility.BrowserFactory.startApplication(BrowserFactory.java:41)
at com.learnautomation.pages.BaseClass.setup(BaseClass.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Trying to start Browser and Getting application ready]]>
            </line>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="SKIP" signature="loginApp()[pri:1, instance:com.learnautomation.testcases.LoginTestCRM@3b084709]" name="loginApp" duration-ms="0" started-at="2019-11-15T18:29:43 IST" finished-at="2019-11-15T18:29:43 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.learnautomation.utility.BrowserFactory.startApplication(BrowserFactory.java:41)
at com.learnautomation.pages.BaseClass.setup(BaseClass.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="SKIP" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.learnautomation.testcases.LoginTestCRM@3b084709]" name="tearDownMethod" is-config="true" duration-ms="-1573822783965" started-at="2019-11-15T18:29:43 IST" finished-at="1970-01-01T05:30:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.learnautomation.testcases.LoginTestCRM@3b084709]" name="tearDown" is-config="true" duration-ms="-1573822783980" started-at="2019-11-15T18:29:43 IST" finished-at="1970-01-01T05:30:00 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.learnautomation.testcases.LoginTestCRM -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
